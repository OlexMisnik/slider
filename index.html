<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">

<head>
	<title>bxSlider</title>
	<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />	
	<link rel="stylesheet" type="text/css" href="css/style.css" />
	<link rel="stylesheet" href="https://cdn.jsdelivr.net/bxslider/4.2.12/jquery.bxslider.css">
	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
	<script src="https://cdn.jsdelivr.net/bxslider/4.2.12/jquery.bxslider.min.js"></script>

	<script>
		$(document).ready(function () {
			$('.slider').bxSlider();
		});
	</script>

</head>

<body>
	<h1>bxSlider</h1>
	<div class="slider">
		<div><img src="images\slide1.jpg" alt=""></div>
		<div><img src="images\slide2.jpg" alt=""></div>
		<div><img src="images\slide3.jpg" alt=""></div>
		<div><img src="images\slide4.jpg" alt=""></div>
	</div>
</body>

</html>