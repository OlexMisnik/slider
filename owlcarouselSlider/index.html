<!DOCTYPE html>
<html lang="en">

<head>
	<title>owlcarouselSlider</title>
	<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
	<link rel="stylesheet" href="css/style.css">
	<link rel="stylesheet" href="css/owl.carousel.min.css">
	<link rel="stylesheet" href="css/owl.theme.default.min.css">

</head>

<body>
	<h1>owlcarouselSLIDER</h1>
	<div class="owl-carousel owl-theme">
		<div class="item"><img src="images\slide1.jpg" alt=""></div>
    <div class="item"><img src="images\slide2.jpg" alt=""></div>
    <div class="item"><img src="images\slide3.jpg" alt=""></div>
    <div class="item"><img src="images\slide4.jpg" alt=""></div>
	</div>
	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
	<script src="js/owl.carousel.min.js"></script>
	<script>
		$(document).ready(function () {
			$('.owl-theme').owlCarousel({
				loop: true,
				margin: 10,
				nav: true,
				responsive: {
					0: {
						items: 1
					},
					600: {
						items: 3
					},
					1000: {
						items: 5
					}
				}
			});
		});
	</script>

</body>

</html>