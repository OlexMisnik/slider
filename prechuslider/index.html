<!DOCTYPE html>
<html lang="ru">

<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <title>prechu</title>
  <link rel="stylesheet" href="css/main.css">
  <link rel="stylesheet" href="css/swiper.min.css">
</head>

<body>

  <!-- <div class="wrapper"> -->


  <header class="section-header">
    <div class="container">
      <div class="section-header-position">
        <div class="section-header-left">
          <div class="logo-img">
            <div class="icon"><img src="images\__8.png" alt=""></div>
          </div>
          <div class="logo-text">Prechu</div>
        </div>
        <div class="section-header-right">
          <nav class="nav">
            <a class="nav-item" href="#">Home</a>
            <a class="nav-item" href="#">Products</a>
            <a class="nav-item" href="#">Services</a>
            <a class="nav-item" href="#">Contact</a>
          </nav>
        </div>
      </div>
    </div>
  </header>




  <div class="swiper-container section-background">
    <div class="swiper-wrapper">
      <div class="swiper-slide"><img class="section-background-img" src="images\dsfsdfsdfds@1X.png" alt="img">></div>
      <div class="swiper-slide"><img class="section-background-img" src="images\michael-liao-TCxqiUCqp8U-unsplash.jpg" alt=""></div>
      <div class="swiper-slide"><img class="section-background-img" src="images\vino-li-O9TUvx3uAww-unsplash.jpg" alt=""></div>
      <div class="swiper-slide"><img class="section-background-img" src="images\zeke-tucker-Vy4lR2NhfX0-unsplash.jpg" alt=""></div>
    </div>

    <div class="swiper-button-next"></div>
    <div class="swiper-button-prev"></div>
  </div>








  <div class="section-frontground">
    <div class="iphone"><img src="images\iphone@1X.png" alt=""></div>
    <div class="title-description">
      <h1 class="title">Lorem Ipsum is simply</h1>
      <p class="description"> dummy text of the<br>
        printing & typesetting </p>
      <div class="button-primary">
        <div class="button-primary-size"> <img class="button-primary-img" src="images\shape_2_14.svg" alt=""></div>
        <div>Sign up now</div>
      </div>
    </div>
  </div>

  </div>
  <div class="container">
    <div class="grid grid-list">
      <div class="grid-item width-1-3 width-1-1-sm grid-mobil">
        <div class="grid-img blue">
          <div class="icon"><img class="blue-size" src="images\shape_3_7.svg" alt=""></div>
        </div>
        <div class="item-text">
          <h2 class="item-title">Contrary to popular</h2>
          <div class="item-descript">Lorem Ipsum is not simply random text. It has roots in a piece </div>
        </div>
      </div>
      <div class="grid-item width-1-3 width-1-1-sm grid-mobil">
        <div class="grid-img red"><img class="red-size" src="images\shape_6_10.svg" alt=""></div>
        <div class="item-text">
          <h2 class="item-title">Contrary to popular</h2>
          <div class="item-descript">Lorem Ipsum is not simply random text. It has roots in a piece </div>
        </div>
      </div>
      <div class="grid-item width-1-3 width-1-1-sm grid-mobil">
        <div class="grid-img yellow"><img class="yellow-size" src="images\shape_5_32.svg" alt=""></div>
        <div class="item-text">
          <h2 class="item-title">Contrary to popular</h2>
          <div class="item-descript">Lorem Ipsum is not simply random text. It has roots in a piece </div>
        </div>
      </div>
    </div>
  </div>
  <div class="container">
    <div class="grid grid-card">
      <div class="grid-item width-1-2 width-1-1-sm card-mobil">
        <h2 class="title-card">The Shopping cart</h2>
        <div class=""><img class="img-card" src="images\cart_36.jpg" alt=""></div>
      </div>

      <div class="grid-item width-1-2 width-1-1-sm card-mobil ">
        <h3 class="title-card">Some Bullet text here</h3>
        <div class="descript-card">Lorem Ipsum is simply dummy text of the printing and typesetting
          Lorem Ipsum has been the industry's standard dummy text ever
          When an unknown printer took a galley of type and scrambled
          It to make a type specimen book. It has survived not only five centuries
          But also the leap into electronic typesetting,
          Remaining essentially unchanged. It was popularised
          The release of Letraset sheets containing Lorem Ipsum passages</div>
      </div>
    </div>
  </div>
  <div class="line"></div>

  <footer class="footer">
    <section class="section-text-footer">Copyright Â© 2012-2013 cssauthor.com
    </section>
  </footer>
  <script src="js/swiper.min.js"></script>

  <!-- Initialize Swiper -->
  <script>
    var swiper = new Swiper('.swiper-container', {
      navigation: {
        nextEl: '.swiper-button-next',
        prevEl: '.swiper-button-prev',
      },
    });
  </script>
  <!-- </div> -->
</body>

</html>