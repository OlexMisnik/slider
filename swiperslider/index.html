<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
  <title>Swiperslider</title>
  <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1, maximum-scale=1">
  <link rel="stylesheet" href="css/style.css">
  <link rel="stylesheet" href="css/swiper.min.css">

  
</head>
<body>  
  <div class="swiper-container">
    <div class="swiper-wrapper">
      <div class="swiper-slide"><img src="images\slide1.jpg" alt=""></div>
      <div class="swiper-slide"><img src="images\slide2.jpg" alt=""></div>
      <div class="swiper-slide"><img src="images\slide3.jpg" alt=""></div>
      <div class="swiper-slide"><img src="images\slide4.jpg" alt=""></div>
    </div>    
    <div class="swiper-pagination"></div>    
    <div class="swiper-button-next"></div>
    <div class="swiper-button-prev"></div>
  </div>

  
  <script src="js/swiper.min.js"></script>

  
  <script>
    var swiper = new Swiper('.swiper-container', {
      slidesPerView: 1,
      spaceBetween: 30,
      loop: true,
      pagination: {
        el: '.swiper-pagination',
        clickable: true,
      },
      navigation: {
        nextEl: '.swiper-button-next',
        prevEl: '.swiper-button-prev',
      },
    });
  </script>
</body>
</html>
