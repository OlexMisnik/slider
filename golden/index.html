<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <link rel="stylesheet" href="css/bootstrap.min.css">
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
    <link rel="stylesheet" href="css/fonts.css">
    <link rel="stylesheet" href="css/style.css">
    <link rel="stylesheet" href="css/media.css">
    <title>Golden </title>
</head>

<body>

    <header class="header">
        <div class="container">
            <div class="row">
                <div class="col-lg-2">
                    <a class="logo-link link" href="#"><img src="img/logo.png" alt="Logo"></a>
                </div>
                <div class="col-lg-9">
                    <ul class="nav d-flex justify-content-end">
                        <li class=" nav-item menu-list"><a class="link" href="#">Home</a></li>
                        <li class="nav-item menu-list"><a class="link" href="#">Services</a></li>
                        <li class="nav-item menu-list"><a class="link" href="#">Portfolio</a></li>
                        <li class="nav-item menu-list"><a class="link" href="#">About</a></li>
                        <li class="nav-item menu-list"><a class="link" href="#">Contact</a></li>
                    </ul>
                </div>
            </div>
        </div>
        <div class="row">
            <div class="col-lg-12 section">
                <p class="subtitle-cover">Welcome to our studio!</p>
                <h1 class="title-cover">It's nice to meet you</h1>
                <button type="button" class="btn btn-primary btn-cover">Tell me more</button>
            </div>
        </div>
    </header>

    <section class="services">
        <div class="container">
            <div class="row">
                <div class="col-lg-12">
                    <h2 class="services-title title">Services</h2>
                    <p class="services-subtitle subtitle">Lorem ipsum dolor sit amet consectetur adipisicing.</p>
                </div>
            </div>
            <div class="row">
                <div class="col-lg-4">
                    <div class="card card-style">
                        <img src="img/icon1.png" class="icon" alt="icon1">
                        <div class="card-body">
                            <h5 class="card-title title-card">E-commerce</h5>
                            <p class="card-text text">Some quick example text to build on the card
                                title and make up the bulk
                                of the card's content.</p>
                        </div>
                    </div>
                </div>
                <div class="col-lg-4">
                    <div class="card card-style">
                        <img src="img/icon2.png" class="icon" alt="icon2">
                        <div class="card-body">
                            <h5 class="card-title title-card">Responsive Web</h5>
                            <p class="card-text text">Some quick example text to build on the card
                                title and make up the bulk
                                of the card's content.</p>
                        </div>
                    </div>
                </div>
                <div class="col-lg-4">
                    <div class="card card-style">
                        <img src="img/icon.png" class="icon" alt="icon3">
                        <div class="card-body">
                            <h5 class="card-title title-card">Web Security</h5>
                            <p class="card-text text">Some quick example text to build on the card
                                title and make up the bulk
                                of the card's content.</p>
                        </div>
                    </div>
                </div>
            </div>
    </section>

    <section class="portfolio">
        <div class="container">
            <div class="row">
                <div class="col-lg-12">
                    <h2 class="services-title title">Our Portfolio</h2>
                    <p class="services-subtitle subtitle">Lorem ipsum dolor sit amet consectetur adipisicing.</p>
                </div>
            </div>
            <div class="row">
                <div class="col-lg-4">
                    <div class="card" style="width: 18rem;">
                        <img src="img/img2.png" class="portfolio-image" alt="icon3">
                        <div class="card-body">
                            <h5 class="card-title title-card">Ebony & Ivory
                            </h5>
                            <p class="card-text text">Branding</p>
                        </div>
                    </div>
                </div>
                <div class="col-lg-4">
                    <div class="card" style="width: 18rem;">
                        <img src="img/img3.png" class="portfolio-image" alt="icon3">
                        <div class="card-body">
                            <h5 class="card-title title-card">Smart Stationary
                            </h5>
                            <p class="card-text text">Print Design</p>
                        </div>
                    </div>
                </div>
                <div class="col-lg-4">
                    <div class="card" style="width: 18rem;">
                        <img src="img/img1.png" class="portfolio-image" alt="icon3">
                        <div class="card-body">
                            <h5 class="card-title title-card">Clever Poster
                            </h5>
                            <p class="card-text text">Print Design</p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

    <section class="contact">
        <div class="container">
            <div class="row">
                <div class="col-lg-12">
                    <h2 class="title" style="color: white">Contact us</h2>
                    <p class="text">Proin iaculis purus consequat sem cure. </p>
                </div>
            </div>

            <form class="form">
                <div class="row">
                    <div class="col">
                        <input type="text" class="form-control input-size" placeholder="your Name *">
                        <input type="email" class="form-control input-size" placeholder="your e-mail *">
                        <input type="text" class="form-control" placeholder="subject">
                    </div>
                    <div class="col">
                        <textarea class="form-control" placeholder="Your message *" rows="6"></textarea>
                    </div>
                </div>
            </form>
            <div class="row">
                <div class="col-lg-12">
                    <button type="button" class="btn btn-primary btn-cover">Send Message</button>
                </div>
            </div>
        </div>
    </section>
  

    <footer class="footer">
        <div class="container">
            <div class="row">
                <div class="col-lg-5">
                    <p class="footer-text text text">&#xa9 Copyrite 2014 <span><a class="link"
                                href="#">FreebiesXpress.com</a></span></p>
                </div>
                <div class="col-lg-7">
                    <div class="social github">
                        <a href="#" target="_blank"><i class="fa fa-github fa-2x"></i></a>
                    </div>
                    <div class="social youtube">
                        <a href="#" target="_blank"><i class="fa fa-youtube fa-2x"></i></a>
                    </div>
                    <div class="social google-pluse">
                        <a href="#" target="_blank"><i class="fa fa-google-plus fa-2x"></i></a>
                    </div>
                    <div class="social facebook">
                        <a href="#" target="_blank"><i class="fa fa-facebook fa-2x"></i></a>
                    </div>
                </div>
            </div>
        </div>
    </footer>


</body>

</html>